src/Xamarin.Android.Tools.AndroidSdk/Xamarin.Android.Tools.Versions.props

```diff
@@ -9,12 +9,14 @@
     If this file is changed the submodule for androidtools should be updated, 
     along with any other repo which references androidtools. 
     -->
-    <AndroidSdkBuildToolsVersion Condition="'$(AndroidSdkBuildToolsVersion)' == ''">30.0.2</AndroidSdkBuildToolsVersion>
-    <AndroidCommandLineToolsVersion Condition=" '$(AndroidCommandLineToolsVersion)' == '' ">2.1</AndroidCommandLineToolsVersion>
-    <AndroidSdkPlatformToolsVersion Condition="'$(AndroidSdkPlatformToolsVersion)' == ''">30.0.4</AndroidSdkPlatformToolsVersion>
-    <AndroidSdkToolsVersion Condition="'$(AndroidSdkToolsVersion)' == ''">26.1.1</AndroidSdkToolsVersion>
+    <AndroidSdkBuildToolsVersion Condition="'$(AndroidSdkBuildToolsVersion)' == ''">31.0.0</AndroidSdkBuildToolsVersion>
+    <AndroidCommandLineToolsVersion Condition=" '$(AndroidCommandLineToolsVersion)' == '' ">5.0</AndroidCommandLineToolsVersion>
+    <AndroidSdkPlatformToolsVersion Condition="'$(AndroidSdkPlatformToolsVersion)' == ''">31.0.3</AndroidSdkPlatformToolsVersion>
     <AndroidSdkEmulatorVersion Condition="'$(AndroidSdkEmulatorVersion)' == ''"></AndroidSdkEmulatorVersion>
-    <AndroidSdkPlatformVersion Condition="'$(AndroidSdkPlatformVersion)' == ''">android-30</AndroidSdkPlatformVersion>
-    <AndroidNdkVersion Condition="'$(AndroidNdkVersion)' == ''">21.3.6528147</AndroidNdkVersion>
+    <AndroidSdkPlatformVersion Condition="'$(AndroidSdkPlatformVersion)' == ''">android-31</AndroidSdkPlatformVersion>
+    <AndroidNdkVersion Condition="'$(AndroidNdkVersion)' == ''">23.0.7599858</AndroidNdkVersion>
```

Comment:
I don't see this version in VS 2022, is there some other place they update this list?

![image](https://user-images.githubusercontent.com/840039/133454011-5e07fd66-6ef3-45bd-a533-f5d328f029ba.png)

Reply:
It appears the Google [repository manifest](https://dl-ssl.google.com/android/repository/repository2-1.xml) is out of date and it doesn't have the [released version](https://developer.android.com/ndk/downloads/index.html#lts-downloads) of the NDK.

Reply:
The latest version the SDK repo has appears to be beta4

Reply:
Maybe we can just put a NDK 22.x for now?